src/Overlay/OverlayTrigger.tsx

```diff
@@ -505,6 +505,15 @@ const OverlayTrigger = React.forwardRef(
         </Overlay>
       );
     };
+
+    if (
+      (typeof children === 'object' && children.type === React.Fragment) ||
+      typeof children === 'string'
+    ) {
+      console.error(
+        '[rsuite]: The OverlayTrigger component does not accept strings or Fragments as child.'
```

Comment:
I would suggest remove the semicolon to be consistent with other warning messages.

```suggestion
        '[rsuite] The OverlayTrigger component does not accept strings or Fragments as child.'
```